src/Core/ViewHelper/AbstractTagBasedViewHelper.php

```diff
@@ -110,7 +110,7 @@ public function initialize()
         }
 
         foreach ($this->additionalArguments as $argumentName => $argumentValue) {
-            if (strpos($argumentName, 'data-') === 0) {
+            if (strpos($argumentName, 'data-') === 0 || strpos($argumentName, 'data-') === 0) {
```

Comment:
This should be `strpos($argumentName, 'aria-')`

Reply:
Yikes, a total rebase failure. :-(